paddle/scripts/submit_local.sh.in

```diff
@@ -112,13 +112,13 @@ fi
 
 case "$1" in
     "train")
-        ${DEBUGGER} $MYDIR/../opt/paddle/bin/paddle_trainer ${@:2}
+        ${DEBUGGER} $MYDIR/../../opt/paddle/bin/paddle_trainer ${@:2}
```

Comment:
@tensor-tang , I tried to modify `data_files` to the absolute path `/usr/local/opt/paddle`, but maybe `date_files` will always be installed under python packages directory,  here is the document from https://docs.python.org/2.7/distutils/setupscript.html#installing-additional-files:
> Each file name in files is interpreted relative to the setup.py script at the top of the package source distribution.